src/specifications/profile-resolution/profile-resolution-specml.xml

```diff
@@ -3,1772 +3,1370 @@
-      includes a <q>no merge option</q>). In source data (a valid profile document), a
-        <src>profile/merge</src> directive is optional; when it or its child directives are absent,
-      default combination and merge rules apply.</p>
-    <p>The <term>modification</term> phase is an opportunity for a profile to amend, alter or edit
-      controls, and to set parameter values or constrain or qualify parameters as represented in the
-      target. A profile can be valid and useful without making any modifications, only selections,
-      so the <src>profile/modify</src> element (node) is optional in the profile model.</p>
     <p>As described in the previous section, when resolved, an OSCAL profile takes the form of an
-      OSCAL catalog. The merge semantics described below will produce outputs conforming to this
-      organization.</p>
-    <section id="import-selections">
-      <head>Selection phase </head>
-      <p>A profile begins by importing catalogs, all of whose controls are made available. The
-        catalogs it references may be available as static resources, or they may be produced
-        dynamically on request; accordingly a profile may also import profiles, to be resolved as
-        catalogs, instead of or in addition to catalogs as such. Imports are given in sequence after
-        the metadata:</p>
-      <tagging whose="source_profile">&lt;profile>
-  &lt;metadata><hi>{ Profile metadata }</hi>&lt;/metadata>
-  &lt;import href="#catalog"><hi>{ … selection of controls … }</hi>&lt;/import>
-  &lt;import href="#catalog-supplemental"><hi>{ … selection of controls … }</hi>&lt;/import>
-&lt;/profile>
-</tagging>
-      <tagging whose="target_catalog">&lt;profile>
-  &lt;metadata><hi>{ Profile metadata }</hi>&lt;/metadata>
-  &lt;selection href="#catalog" catalog-id="<hi>{ catalog-identifier }</hi>"><hi>{ … selection of controls … }</hi><hi>{{ end selection }}</hi>
-  &lt;selection href="#catalog-supplemental" catalog-id="<hi>{ catalog-identifier }</hi>"><hi>{ … selection of controls … }</hi><hi>{{ end selection }}</hi>
-&lt;/profile>
-</tagging>
-      <p>The resource indicated can be either an OSCAL catalog or an OSCAL profile. A catalog
-        provides controls in their native form. An imported profile is resolved on import, using the
-        same rules for profile resolution, so a catalog is presented to the importing profile.</p>
-      <p>(The result of importing other endpoints or resources besides valid OSCAL catalogs and
-        profiles is not defined by this specification.)</p>
-      <p>No profile may import itself either directly or indirectly. An import directive that
-        indicates either the profile itself, or a profile into which it is being (currently)
-        imported, must be ignored. Optionally, a processor may issue a warning.</p>
+      OSCAL catalog. The phases described below will produce outputs conforming to this model.</p>
```

Comment:
```suggestion
      OSCAL catalog. The phases described below will produce outputs conforming to the catalog model.</p>
```

There is no guarantee that the produced catalog will be valid or conform to the catalog model. In fact, there are plenty of cases were the actions of tailoring might produce invalid catalog content. Perhaps this should be discussed here?